eo-runtime/src/main/eo/org/eolang/collections/list.eo

```diff
@@ -95,7 +95,29 @@
         &.f x > @
 
   # For each array element dataize the object
-  [f] > each /bool
+  # Here "f" must be an abstract object with
+  # one free attribute, the element of the
+  # array.
+  [f] > each
+    if. > @
```

Comment:
@l3r8yJ This check is redundant, because if the array is empty, the while loop will do nothing. And I think there is no need to return `FALSE` at all.

Reply:
@l3r8yJ please, read [this](https://arxiv.org/abs/2111.13384): `9 Key Features:` `No NULL References` and `No Empty Objects`